<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyOutpost</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cOpen">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cArchive">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Archive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cDelete">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cPrint">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cSendReceive">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>10</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Send/Receive</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cProfile">
    <property name="geometry">
     <rect>
      <x>1055</x>
      <y>10</y>
      <width>196</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>985</x>
      <y>10</y>
      <width>66</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Profile:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>140</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Folder List</string>
    </property>
    <widget class="QPushButton" name="cInTray">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>120</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>In Tray</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cOutTray">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>120</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Out Tray</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cSentMessages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>120</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Sent Messages</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cArchived">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>120</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Archive</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cDrafMessages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>120</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Draft Messages</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cDeleted">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>120</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Deleted</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cFolder1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Folder 1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cFolder2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Folder 2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cFolder3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Folder 3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cFolder4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Folder 4</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cFolder5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Folder 5</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cFolderSearchResults">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Search Results</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="cMailList">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>65</y>
      <width>1070</width>
      <height>481</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>U</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>From</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>To</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>BBS</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Local ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Subject</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date/Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="cSendOnly">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>SendOnly</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cReceiveOnly">
    <property name="geometry">
     <rect>
      <x>865</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Receive Only</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cSearch">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1276</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuMove_to_Folder">
     <property name="title">
      <string>Move to Folder</string>
     </property>
     <addaction name="actionIn_Tray"/>
     <addaction name="actionOut_Tray"/>
    </widget>
    <widget class="QMenu" name="menuCopy_to_Folder">
     <property name="title">
      <string>Copy to Folder</string>
     </property>
     <addaction name="actionIn_Tray_2"/>
     <addaction name="actionOut_Tray_2"/>
    </widget>
    <addaction name="actionNew_Message"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_As_No_Headers"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="actionImport"/>
    <addaction name="menuMove_to_Folder"/>
    <addaction name="menuCopy_to_Folder"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_All_Messages"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_No_Headers"/>
    <addaction name="actionPrint_Setup"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <widget class="QMenu" name="menuProfiles">
     <property name="title">
      <string>Profiles</string>
     </property>
     <addaction name="actionNewProfile"/>
     <addaction name="actionSaveProfile"/>
     <addaction name="actionDeleteProfile"/>
    </widget>
    <addaction name="actionBBS"/>
    <addaction name="actionInterface"/>
    <addaction name="actionStation_ID"/>
    <addaction name="mSetupAddress_Book"/>
    <addaction name="menuProfiles"/>
    <addaction name="actionComputer_Time_Check"/>
    <addaction name="separator"/>
    <addaction name="actionReset_all_to_SCC_standard"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuScripts">
     <property name="title">
      <string>Scripts</string>
     </property>
     <addaction name="actionScripting_Tool"/>
     <addaction name="actionPick_and_Run_a_Script"/>
    </widget>
    <widget class="QMenu" name="menuInteractive_Packet">
     <property name="title">
      <string>Interactive Packet</string>
     </property>
     <addaction name="actionSerial_Com_Port"/>
     <addaction name="actionAGW_Packet_Engine"/>
     <addaction name="actionTelnet"/>
    </widget>
    <addaction name="actionSend_Receive_Settings"/>
    <addaction name="actionMessage_Settings"/>
    <addaction name="actionReport_Settings"/>
    <addaction name="actionLog_Settings"/>
    <addaction name="actionGeneral_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionRun_at_Startup_Settings"/>
    <addaction name="actionScript_Settings"/>
    <addaction name="menuScripts"/>
    <addaction name="separator"/>
    <addaction name="actionPsclet_Session_Counter_Reset"/>
    <addaction name="separator"/>
    <addaction name="actionReset_Column_Widths"/>
    <addaction name="actionPack_Column_Widths"/>
    <addaction name="actionReset_Form_to_Normal"/>
    <addaction name="separator"/>
    <addaction name="menuInteractive_Packet"/>
   </widget>
   <widget class="QMenu" name="menuForms">
    <property name="title">
     <string>Forms</string>
    </property>
    <addaction name="actionICS_309_Communications_Log"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="separator"/>
    <addaction name="actionSend_Receive"/>
    <addaction name="actionSend_Receive_No_Bulletins"/>
    <addaction name="actionSend_Only"/>
    <addaction name="actionReceive_Only"/>
    <addaction name="separator"/>
    <addaction name="actionSet_to_Unread"/>
    <addaction name="actionSet_to_Read"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Message_2"/>
    <addaction name="actionOoen_a_Message"/>
    <addaction name="actionOpen_Enhanced_Message_in_Client"/>
    <addaction name="actionOpen_Enhanced_Message_as_Text"/>
    <addaction name="separator"/>
    <addaction name="actionForce_One_Time_Bulletin_Retreive"/>
    <addaction name="separator"/>
    <addaction name="actionView_Outpost_Data_Directories"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOutpost_User_QuickStart"/>
    <addaction name="actionOutpost_User_Basics"/>
    <addaction name="actionOutpost_Scripting"/>
    <addaction name="actionOutpost_ICS213_Messaging_User_Guide"/>
    <addaction name="separator"/>
    <addaction name="actionMy_IP_Address"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Outpost"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSetup"/>
   <addaction name="menuTools"/>
   <addaction name="menuForms"/>
   <addaction name="menuActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="cStatusBar"/>
  <action name="actionBBS">
   <property name="text">
    <string>BBS...</string>
   </property>
  </action>
  <action name="actionInterface">
   <property name="text">
    <string>Interface...</string>
   </property>
  </action>
  <action name="actionStation_ID">
   <property name="text">
    <string>Station ID...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="mSetupAddress_Book">
   <property name="text">
    <string>Address Book...</string>
   </property>
  </action>
  <action name="actionComputer_Time_Check">
   <property name="text">
    <string>Computer Time Check</string>
   </property>
  </action>
  <action name="actionNewProfile">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionSaveProfile">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionDeleteProfile">
   <property name="text">
    <string>Delete...</string>
   </property>
  </action>
  <action name="actionNew_Message">
   <property name="text">
    <string>New Message</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSave_As_No_Headers">
   <property name="text">
    <string>Save As, No Headers...</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionDelete_All_Messages">
   <property name="text">
    <string>Delete All Messages</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print...</string>
   </property>
  </action>
  <action name="actionPrint_No_Headers">
   <property name="text">
    <string>Print, No Headers...</string>
   </property>
  </action>
  <action name="actionPrint_Setup">
   <property name="text">
    <string>Print Setup...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>SelectAll</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSend_Receive_Settings">
   <property name="text">
    <string>Send/Receive Settings...</string>
   </property>
  </action>
  <action name="actionMessage_Settings">
   <property name="text">
    <string>Message Settings...</string>
   </property>
  </action>
  <action name="actionReport_Settings">
   <property name="text">
    <string>Report Settings...</string>
   </property>
  </action>
  <action name="actionLog_Settings">
   <property name="text">
    <string>Log Settings...</string>
   </property>
  </action>
  <action name="actionGeneral_Settings">
   <property name="text">
    <string>General Settings...</string>
   </property>
  </action>
  <action name="actionRun_at_Startup_Settings">
   <property name="text">
    <string>Run at Startup Settings...</string>
   </property>
  </action>
  <action name="actionScript_Settings">
   <property name="text">
    <string>Script Settings...</string>
   </property>
  </action>
  <action name="actionScripting_Tool">
   <property name="text">
    <string>Scripting Tool</string>
   </property>
  </action>
  <action name="actionPick_and_Run_a_Script">
   <property name="text">
    <string>Pick and Run a Script</string>
   </property>
  </action>
  <action name="actionPsclet_Session_Counter_Reset">
   <property name="text">
    <string>Packet Session Counter Reset</string>
   </property>
  </action>
  <action name="actionReset_Column_Widths">
   <property name="text">
    <string>Reset Column Widths</string>
   </property>
  </action>
  <action name="actionPack_Column_Widths">
   <property name="text">
    <string>Pack Column Widths</string>
   </property>
  </action>
  <action name="actionReset_Form_to_Normal">
   <property name="text">
    <string>Reset Form to Normal</string>
   </property>
  </action>
  <action name="actionSerial_Com_Port">
   <property name="text">
    <string>Serial / Com Port</string>
   </property>
  </action>
  <action name="actionAGW_Packet_Engine">
   <property name="text">
    <string>AGW Packet Engine</string>
   </property>
  </action>
  <action name="actionTelnet">
   <property name="text">
    <string>Telnet</string>
   </property>
  </action>
  <action name="actionSend_Receive">
   <property name="text">
    <string>Send/Receive</string>
   </property>
  </action>
  <action name="actionSend_Receive_No_Bulletins">
   <property name="text">
    <string>Send/Receive, No Bulletins</string>
   </property>
  </action>
  <action name="actionSend_Only">
   <property name="text">
    <string>Send Only</string>
   </property>
  </action>
  <action name="actionReceive_Only">
   <property name="text">
    <string>Receive Only</string>
   </property>
  </action>
  <action name="actionSet_to_Unread">
   <property name="text">
    <string>Set to Unread</string>
   </property>
  </action>
  <action name="actionSet_to_Read">
   <property name="text">
    <string>Set to Read</string>
   </property>
  </action>
  <action name="actionNew_Message_2">
   <property name="text">
    <string>New Message</string>
   </property>
  </action>
  <action name="actionOoen_a_Message">
   <property name="text">
    <string>Open a Message</string>
   </property>
  </action>
  <action name="actionOpen_Enhanced_Message_in_Client">
   <property name="text">
    <string>Open Enhanced Message in Client</string>
   </property>
  </action>
  <action name="actionOpen_Enhanced_Message_as_Text">
   <property name="text">
    <string>Open Enhanced Message as Text</string>
   </property>
  </action>
  <action name="actionForce_One_Time_Bulletin_Retreive">
   <property name="text">
    <string>Force One-Time Bulletin Retreive</string>
   </property>
  </action>
  <action name="actionView_Outpost_Data_Directories">
   <property name="text">
    <string>View Outpost Data Directories</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionOutpost_User_QuickStart">
   <property name="text">
    <string>Outpost User QuickStart</string>
   </property>
  </action>
  <action name="actionOutpost_User_Basics">
   <property name="text">
    <string>Outpost User Basics</string>
   </property>
  </action>
  <action name="actionOutpost_Scripting">
   <property name="text">
    <string>Outpost Scripting</string>
   </property>
  </action>
  <action name="actionOutpost_ICS213_Messaging_User_Guide">
   <property name="text">
    <string>Outpost ICS213 Messaging User Guide</string>
   </property>
  </action>
  <action name="actionMy_IP_Address">
   <property name="text">
    <string>My IP Address</string>
   </property>
  </action>
  <action name="actionAbout_Outpost">
   <property name="text">
    <string>About Outpost</string>
   </property>
  </action>
  <action name="actionReset_all_to_SCC_standard">
   <property name="text">
    <string>Reset all to SCC standard</string>
   </property>
  </action>
  <action name="actionIn_Tray">
   <property name="text">
    <string>In Tray</string>
   </property>
  </action>
  <action name="actionOut_Tray">
   <property name="text">
    <string>Out Tray</string>
   </property>
  </action>
  <action name="actionIn_Tray_2">
   <property name="text">
    <string>In Tray</string>
   </property>
  </action>
  <action name="actionOut_Tray_2">
   <property name="text">
    <string>Out Tray</string>
   </property>
  </action>
  <action name="actionXSC_ICS_213_Message">
   <property name="text">
    <string>XSC ICS-213 Message</string>
   </property>
  </action>
  <action name="actionXSC_Damage_Assessment">
   <property name="text">
    <string>XSC Damage Assessment</string>
   </property>
  </action>
  <action name="actionXSC_Check_In_Out_Message">
   <property name="text">
    <string>XSC Check-In/Out Message</string>
   </property>
  </action>
  <action name="actionICS_309_Communications_Log">
   <property name="text">
    <string>ICS-309 Communications Log</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
